src/css/components/loading.scss

```diff
@@ -84,3 +84,12 @@ $indicator-size: 3.5rem;
     opacity: .2;
   }
 }
+
+.saving.saving-relative {
```

Comment:
This was based on the previous styles of `loading-relative`. I believe the `-relative` references the position property.